### ISSUE TITLE
Replace Redundant 'Company-Box' Prefixes

### ISSUE THREAD

**Issue by nealot** [sebastiencs/company-box#22]
At the moment, using company-box alongside company requires two seperate variables for certain faces/variables/keybindings. I don't think this is necessary, as company-box is only a frontend for company-box and is mutually exclusive to the other tooltip frontends. As such, it would be more convienient to remove the company-box variables that simply inherit their values from tooltip and just use the original tooltip variables. This offers a better integration platform for company and would make your job as maintainer much simpler since you don't have to reimplement everything.

I think renaming the variables could improve coherency if company-box also included a general backend but as it isn't trying to replace company itself, this view might only add complexity.

For example, why does ```company-box-align-annotations``` exist? Does it need to be a seperate variable or can we just use ```company-tooltip-align-annotations```?

**Comment by sebastiencs** [comment]
Yes I shouldn't have introduced similar variables, they need to be deleted.
PR are welcome :)
